version

```diff
@@ -1 +1 @@
-0.7.3
+0.8.0
```

Comment:
Isn't it a bugfix? Why not 0.7.4?

Reply:
Cause this release includes the param_raw addition.